protocols/swap/MIGRATION.md

```diff
@@ -2,7 +2,7 @@
 
 This document is intended for those migrating from Swap V1 to V2 and does not include all of the features of Swap V2. [Read more about Swap V2](README.md).
 
-:bulb: **Note**: A key feature of V2 is that every order must have a `nonce` value that is unique to its makerWallet.
+:bulb: **Note**: A key feature of V2 is that every order must have a `nonce` value that is unique to its signerWallet.
 
 ## Signing an Order
```

Comment:
The arguments in the below code excerpt still use maker/taker not signer/sender

Reply:
That's fine since the first example focuses on V1.